social/src/routes/twitter/home-timeline.ts

```diff
@@ -0,0 +1,48 @@
+import express, { Request, Response } from 'express';
+import {
+  BadRequestError,
+  FailedConnectionError,
+  ForbiddenError,
+  requireAuth,
+} from '@tcosmin/common';
+import twit from 'twit';
+import { UserController } from '../../controllers/user-controller';
+
+const router = express.Router();
+const consumerKey = process.env.TWITTER_CONSUMER_KEY!;
+const consumerSecret = process.env.TWITTER_CONSUMER_SECRET!;
```

Comment:
I believe you have a .env file in your local repository, I would suggest create a new file in the root path of your project called .env.example where you have all the keys without their values. Reason being that everyone who sees you repo will know what keys are necessary in order to run the project.

Reply:
Env variables are set inside every microservice config file (X-depl.yaml). These variables are set inside the Kubernetes cluster and microservices will throw an error at startup if one variable is not set

Reply:
Maybe I can list these variables as well as how to set one inside the Kubernetes cluster inside a "How to run the app" section in README.md at a later date

Reply:
Would be a good thing to have a README for the entire project.